java/ql/src/experimental/Security/CWE/CWE-094/JexlInjectionLib.qll

```diff
@@ -0,0 +1,288 @@
+import java
+import semmle.code.java.dataflow.FlowSources
+import semmle.code.java.dataflow.TaintTracking
+
+/**
+ * A taint-tracking configuration for unsafe user input
+ * that is used to construct and evaluate a Jexl expression.
+ * It supports both Jexl2 and Jexl3.
+ */
+class JexlInjectionConfig extends TaintTracking::Configuration {
+  TaintPropagatingJexlMethodCall taintPropagatingJexlMethodCall;
```

Comment:
```suggestion
```

Using this as a field generates multiple instances of the Configuration. I'm not sure the consequences of that, but they surely can't be good :)

Reply:
I didn't know about it, thanks! I'll apply this suggestion.